api/backend.go

```diff
@@ -304,12 +306,12 @@ func (b *StatusBackend) registerHandlers() error {
 			return nil, err
 		}
 
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
+		result := b.SendTransaction(ctx, txArgs, "", nil)
```

Comment:
What is this SendTransaction call for? Not sure what arguments I should use here.

Reply:
this one is from RPC, if Clojure intercepts everything you don't need this `RegisterHandler` it at all
/cc @flexsurfer